src/features/WriteStoriesModal/code/typescript.tsx

```diff
@@ -23,7 +23,11 @@ export default [
     },
   ],
   [
-    { code: `export const Primary: Story = {` },
+    {
+      code: `type Story = StoryObj<Button>;
```

Comment:
please change the nextjs-typescript as well

Reply:
Done ✅